packages/dcos-integration-test/extra/test_applications.py

```diff
@@ -90,7 +90,7 @@ def test_octarine_http(cluster, timeout=30):
         }]
     }
 
-    cluster.deploy_marathon_app(app_definition)
+    cluster.marathon_deploy_and_cleanup(app_definition)
```

Comment:
/me thought the octarine server + client test talked to eachother. Since it seems not, this works.